examples/moodle-presslabs-stash/steps.txt

```diff
@@ -0,0 +1,167 @@
+Setup and Deploy KubePlus
+-----
+
+0) Install aws-cli, entering your access-keys
+    - curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+    - unzip awscli-bundle.zip
+    - sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+    - aws configure
+
+1) Make an s3 bucket
+    - aws s3 mb s3://bucket_name_here1234
+
+2) Start Minikube
+    - minikube start --memory 4096 --cpus=2
+
+3) Allow local images
+    - eval $(minikube docker-env)
+
+4) Go to kubediscovery location
+    - cd ../kubediscovery
+
+5) Build kubediscovery image that supports namespace and has bug fixes
+    - ./build-local-discovery-artifacts.sh
+
+6) Go back to kubeplus
+    - cd ../kubeplus
+
+7) Edit kubeplus/deploy/rc.yaml Line 30, to use local image:
+    - image: kube-discovery-apiserver:latest
+
+8) Deploy KubePlus pod
+    - kubectl apply -f deploy/
+
+
+Start helm and install operators
+-----------
+
+1) Setup
+    - helm init
+
+2) Wait till Tiller Pod is running
+    - kubectl get pods -n kube-system
+
+3) Install presslabs-mysql-operator  and stash operator(note openapispec.json needs to be added to deploy with kubeplus)
+
+helm install https://github.com/cloud-ark/operatorcharts/blob/master/mysql-operator-0.2.5.tgz?raw=true
+helm install https://github.com/cloud-ark/operatorcharts/blob/master/stash-operator-chart-0.8.3.tgz?raw=true
+
+4) Change directory
+    - cd examples/moodle-presslabs-stash
+
+5) Deploy moodle operator
+    - kubectl apply -f moodle-operator.yaml
```

Comment:
I see. You are deploying moodle-operator using KubePlus whereas other two Operators using Helm. Cool.